client/src/i18n/en/cashbox.json

```diff
@@ -9,6 +9,17 @@
     "DESCRIPTION":"This module is responsible for cashbox management",
     "TITLE":"Cashbox Management",
     "NO_SUPPORTED_CURRENCIES":"This cashbox does not have any supported currencies.",
-    "SET_CURRENCY_ACCOUNTS":"Set currency accounts for this cashbox."
+    "SET_CURRENCY_ACCOUNTS":"Set currency accounts for this cashbox.",
+
+    "USERS": { 
+      "TITLE": "Cashbox Users",
+      "SUBTITLE": "Users",
+      "INFO_1": "To update users with access to this cash box, use the",
```

Comment:
Just FYI, it's possible to include HTML tags in these translation values, as well as variable substitutions.  An example of that is [in patient records](https://github.com/IMA-WorldHealth/bhima-2.X/blob/master/client/src/i18n/en/patient_records.json#L43).  I'm pretty sure you could even include a `ui-sref` with the addition of a [translate-compile](https://angular-translate.github.io/docs/#/guide/05_using-translate-directive#using-translate-directive_post-compiling) tag.

Personally, I prefer this method (even though it seems impure) since it allows you to preserve the context of the statement and allow developers to infer from the string what is going to happen.  It would look something like this for your code:
```js
"INFO" : "To update users with access to this cash box, use the <a href ui-sref="users.list">Users Management</a> link."
```

What do you think?

Reply:
Of course you'll need to then have `translate-compile` on the `translate` directive tag in your HTML.

Reply:
I would also add "The following users have access to this cashbox:" at the very end, just to be explicit.

Reply:
This is good to know about! I will make follow this suggestion for this example; I think the context of what is being translated is lost slightly when broken into 3 pieces. 

In general though I think that it's concerning to put client application logic in i18n translation files. In this case it's fairly simple, I think any use of `<b>` or `<u>` or general style changes should be fine. Potentially we should be more careful when it comes to including elements that change the way the application works.

```html
<span translate translate-compile translate-sanitize-strategy="'sce'">
    CASHBOX.USERS.INFO_LINK
</span> 
``` 

Does read clearer than the previous implementation, however.